docs/api/chrome-command-line-switches.md

```diff
@@ -55,6 +55,8 @@ $ electron --js-flags="--harmony_proxies --harmony_collections" your-app
 
 See the [Node documentation][node-cli] or run `node --help` in your terminal for a list of available flags. Additionally, run `node --v8-options` to see a list of flags that specifically refer to Node's V8 JavaScript engine.
 
+> **NOTE**: This only affects the JavaScript environment of the browser process. Renderer processes are not affected by this flag.
```

Comment:
```suggestion
> **NOTE**: This only affects the JavaScript environment of the main process. Renderer processes are not affected by this flag.
```

TIL "browser process" vs "[main process](https://github.com/electron/electron/blob/8ba271efba691357ab1970c1a553b4e019125574/docs/glossary.md#main-process)"

Reply:
Ah, I didn't realise that Electron had changed the term. 👍